designsystem/package.json

```diff
@@ -21,7 +21,8 @@
     "build": "make all",
     "start": "snowpack dev",
     "ghpages": "snowpack build --polyfill-node",
-    "clean": "make clean"
+    "clean": "make clean",
+    "test": "npx playwright test"
```

Comment:
Er der en spesiell grunn til at `playwright` ikke ligger inne i devDependencies men blir dratt inn med `npx`?

Reply:
Ikke annet enn at det var sånn de brukte det i dokumentasjonen. Skjønner ikke helt hva du mener med at playwright ikke ligger inne i dev dependencies? For det er jo lista opp der? Men jeg får ikke brukt npm run playwright, eller noen playwright kommandoer, så npx var det eneste som jeg fikk til å funke

Reply:
Noen forslag til hvordan jeg kan løse dette? eller kan jeg la det stå?